package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "zen-audio-player-electron",
+  "version": "0.1.0",
+  "main": "main.js",
+  "dependencies": {
+    "electron-prebuilt": "^1.2.7"
+    },
+   "scripts": {
+    "pre-lint": "npm install eslint",
```

Comment:
we shouldn't need this `pre-lint` step if we add `eslint` under `devDependencies`